build/configs/rtl8721csm/hello/defconfig

```diff
@@ -188,7 +190,7 @@ CONFIG_BOARD_LOOPSPERMSEC=22222
 #
 CONFIG_ARCH_HAVE_INTERRUPTSTACK=y
 CONFIG_ARCH_INTERRUPTSTACK=0
-# CONFIG_ARCH_HAVE_NESTED_INTERRUPT is not set
+CONFIG_ARCH_HAVE_NESTED_INTERRUPT=y
```

Comment:
Do we need to use this config?

Reply:
it's auto generated. after I enter the menuconfig. I didn't set it manually.